badgecheck/tasks/validation.py

```diff
@@ -211,7 +233,7 @@ def __init__(self, class_name):
                 #   'expected_class': OBClasses.VerificationObject, 'required': True},
                 {'prop_name': 'issuedOn', 'prop_type': ValueTypes.DATETIME, 'required': True},
                 {'prop_name': 'expires', 'prop_type': ValueTypes.DATETIME, 'required': False},
-                {'prop_name': 'image', 'prop_type': ValueTypes.URL, 'required': False},  # TODO: ValueTypes.DATA_URI_OR_URL
+                {'prop_name': 'image', 'prop_type': ValueTypes.DATA_URI_OR_URL, 'required': False},
```

Comment:
Looks good other than this one line. We'll actually need to make one minor change once we have some real test data that the Assertion.image property may not be a data URI (I was incorrect in my comment). As the Assertion.image is defined to be an image that embeds the Assertion, we can't circularly allow the image to be embedded in the Assertion.